docs/realm.js

```diff
@@ -499,6 +499,33 @@ class Realm {
  *   name. Queries can be done using both the public and the underlying property name.
  */
 
+/**
+ * The retreived schema from an open Realm has a different structure as the input schema.
```

Comment:
```suggestion
 * The retreived schema from an open Realm might have a different structure as the input schema.
```

Reply:
:shipit: We rarely use the canonical schema notation in examples

Reply:
This change would denote that it also might return ObjectSchema?  I want to show that this isn't the case.  `realm.schema[n]` always returns a CanonicalObjectSchema.